pkg/gotemplate/options.go

```diff
@@ -308,6 +308,41 @@ Options:
 					},
 				},
 			},
+			{
+				Name: "ci",
+				Options: []Option{
+					{
+						name:         "provider",
+						defaultValue: StaticValue(1),
```

Comment:
Should 1 be the default value or rather no ci? 🤔

Reply:
I think the most commonly used provider is GitHub and for that would it be nice to have GitHub Actions as default.

Reply:
alright